osu.Game/Screens/Play/PauseOverlay.cs

```diff
@@ -24,13 +26,13 @@ public class PauseOverlay : GameplayMenuOverlay
         protected override Action BackAction => () => InternalButtons.Children.First().Click();
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours, AudioManager audio)
+        private void load(OsuColour colours, AudioManager audio, SkinManager skins)
         {
             AddButton("Continue", colours.Green, () => OnResume?.Invoke());
             AddButton("Retry", colours.YellowDark, () => OnRetry?.Invoke());
             AddButton("Quit", new Color4(170, 27, 39, 255), () => OnQuit?.Invoke());
 
-            var sampleChannel = audio.Samples.Get(@"Gameplay/pause-loop");
+            var sampleChannel = skins.GetSample(new SampleInfo("pause-loop")) ?? audio.Samples.Get(@"Gameplay/pause-loop");
```

Comment:
Changing skins will break this. If you check how other components have a skinnable sample playing (e.g. `NightcoreBeatContainer`), you'll find they're using the `SkinnableSound` component, put that in place of `DrawableSample`.

Though now you won't have access to the audio properties transforms, I would say write ```csharp
pauseLoop?.TransformBindableTo(pauseLoop.Volume, ...);
```
in replace to that for now.